chrome/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "Libor Polčák, Martin Bednář, Martin Timko, Pavel Pohner, Peter Horňák",
   "background": {
-      "scripts": ["helpers.js", "update.js", "url.js", WRAPPING, "code_builders.js", "levels.js", "background.js", "level_cache.js", "http_shield_chrome.js", "http_shield_common.js"],
+      "scripts": ["sessionHash.js", "helpers.js", "update.js", "url.js", WRAPPING, "code_builders.js", "levels.js", "background.js", "level_cache.js", "http_shield_chrome.js", "http_shield_common.js"],
```

Comment:
You'll probably need to move the script after helpers.js, see below.

Also looking at the other filenames, session_hash.js looks more consistent.